multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -389,3 +402,28 @@ def infer_problem_type_output_shape(
                 f"The label column '{label_column}' has type"
                 f" '{column_types[label_column]}', which is not supported yet."
             )
+
+
+def set_fallback_column_type(column_types: Dict, allowable_column_types: List[str], fallback_column_type: str) -> Dict:
```

Comment:
How to use this function? In particular, how to set `fallback_column_type` based on user's input?

Reply:
See the new util function `infer_dtypes_by_model_names`. The `allowable_column_types ` can be obtained from config. The `fallback_column_type ` is some prior knowledge we can set based on user cases. Here we have the case of `image` and `text`. Since `image` is easy to detect, we can change other detected types all to `text`. We can add more rules in `infer_dtypes_by_model_names`.